packages/web-components/src/components/content-switcher/content-switcher.mdx

```diff
@@ -73,27 +71,27 @@ You can disable the content switcher items by adding the `disabled` attribute to
 `cds-content-switcher-item` elements directly.
 
 <cds-content-switcher>
-  <cds-content-switcher-item disabled name="one" text="First section">
-    Option 1
+  <cds-content-switcher-item disabled value="all" name="one" text="First section">
```

Comment:
looks like content-switcher-item doesn't have a `text` attribute and just uses the text inside that gets slotted. This should be removed in all examples code not just here
```suggestion
  <cds-content-switcher-item disabled value="all" name="one" >
```